ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -82,6 +77,39 @@
       />
     </Hds::ButtonSet>
   </div>
+  {{#if this.showMultipleModalsMessage}}
+    <Hds::Modal
+      id="multiple-message-modal"
+      @size="large"
+      @color="warning"
+      @onClose={{fn (mut this.showMultipleModalsMessage) false}}
+      data-test-modal="preview modal"
+      as |M|
+    >
+      <M.Header data-test-modal-title="Warning: more than one modal">
+        Warning: more than one modal
+      </M.Header>
+      <M.Body data-test-modal-body="Warning: more than one modal">
+        You have an active modal configured and are trying to create another one. It is recommended to
+        <b>avoid having more than one modal</b>
+        at once as it can be intrusive for users. Would you like to continue creating your message? Click “Confirm” to
+        continue.
+      </M.Body>
+      <M.Footer>
+        <Hds::Button
+          @text="Confirm"
+          {{on "click" (fn this.updateUserConfirmation "confirmed")}}
+          data-test-modal-button="confirm"
+        />
+        <Hds::Button
+          @text="Cancel"
+          @color="secondary"
+          {{on "click" (fn this.updateUserConfirmation "")}}
```

Comment:
We'll want to update this to some falsy value so that we exit the loop on line 79 of the backing class
```suggestion
          {{on "click" (fn this.updateUserConfirmation "cancel")}}
```